template.ipynb

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
<p>Need to supply rules on formatting like only use one H1 (# Title) or else it will break formatting for other outputs (like toctree in Sphinx).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1570/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>